apps/src/sites/studio/pages/layouts/_small_footer.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import getScriptData from '@cdo/apps/util/getScriptData';
+import SmallFooter from '@cdo/apps/code-studio/components/SmallFooter';
+
+// Note: We're not waiting for document.ready; we expect this script to be inlined into
+// the DOM immediately after the necessary #page-small-footer markup.
+ReactDOM.render(
+  <SmallFooter {...getScriptData('smallfooter')} />,
```

Comment:
Note, for the data attribute I'm stringing together words `scriptdata` instead of separating them as I normally would `script-data/script_data/scriptData` because when HAML renders a data-attribute it tries to do a smart transformation of multi-word names... which, in my opinion, is surprising behavior and generally hurts searchability when trying to locate the source or eventual use of a particular data attribute.

Reply:
➕   this drives me wild about data-attributes/HAML! good call to just keep it one word.